### PR TITLE
Py3 Upgrades

### PR DESCRIPTION
**Pull request by alexfikl**
Ran [pyupgrade](https://github.com/asottile/pyupgrade) over everything.

### CODE REVIEW

File: test/extra_int_eq_data.py [hunk 1]
```diff
@@ -543,61 +543,61 @@ def get_mesh(self, resolution, target_order):
...
-                    Field[7].FieldsList = {2, 4, 6};
+                    Field[7].FieldsList = {{2, 4, 6}};
 
                     Background Field = 7;
-                    """ % {
-                        "lcmax": resolution,
-                        }, ["betterplane.brep"]), 2)
+                    """.format(
+                        lcmax=resolution,
+                        ), ["betterplane.brep"]), 2)
```
Comment (inducer) [inline test/extra_int_eq_data.py:600]: No like.

File: pytential/symbolic/mappers.py [hunk 2]
```diff
@@ -600,31 +596,31 @@ def map_ones(self, expr, enclosing_prec):
         return "Ones[%s]" % stringify_where(expr.dofdesc)
 
     def map_inverse(self, expr, enclosing_prec):
-        return "Solve(%s = %s {%s})" % (
+        return "Solve({} = {} {{{}}})".format(
```
Comment (inducer) [inline pytential/symbolic/mappers.py:599]: Meh.

Comment (inducer) [reply pytential/symbolic/mappers.py]: d9f8f97

File: test/extra_int_eq_data.py (hunk 1 above)
Comment (inducer) [reply test/extra_int_eq_data.py]: d9f8f97

File: pytential/symbolic/mappers.py (hunk 2 above)
Comment (alexfikl) [reply pytential/symbolic/mappers.py]: Oh no, I missed that! They changed my python into lisp!

**Comment by inducer** [comment]
Thanks for working on this! Ready to land once CI passes.

**Comment by alexfikl** [comment]
> Thanks for working on this! Ready to land once CI passes.

I was just about to go fix those. Thanks for the help!

**Pull request merged**